lib/ansible/modules/lineinfile.py

```diff
@@ -44,6 +44,18 @@
     type: str
     aliases: [ regex ]
     version_added: '1.7'
+  literal:
+    description:
+      - The literal or raw expression to look for in every line of the file.
+      - For C(state=present), the pattern to replace if found. Only the last line found will be replaced.
+      - For C(state=absent), the pattern of the line(s) to remove.
+      - If the literal expression is not matched, the line will be
+        added to the file in keeping with C(insertbefore) or C(insertafter)
+        settings.
+      - When modifying a line the literal should typically match both the initial state of
+        the line as well as its state after replacement by C(line) to ensure idempotence.
```

Comment:
The literal doesn't need to match both initial and replaced state of the line and would not be possible in many cases. If no match is found, the line will be looked for literally already.
```suggestion
      - The literal string to look for in every line of the file. This does not have to match the entire line.
      - For C(state=present), the line to replace if the string is found in the filen. Only the last line found will be replaced.
      - For C(state=absent), the line(s) to remove if the string is in the line.
      - If the literal expression is not matched, the line will be
        added to the file in keeping with C(insertbefore) or C(insertafter)
        settings.
      - Mutually exclusive with C(backrefs) and C(regexp).
```